test/bats/akeyless.bats

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bats
+
+ load helpers
+
+ BATS_TESTS_DIR=test/bats/tests/akeyless
+ WAIT_TIME=120
+ SLEEP_TIME=1
+
+
+ export LABEL_VALUE=${LABEL_VALUE:-"test"}
+ export ANNOTATION_VALUE=${ANNOTATION_VALUE:-"app=test"}
+
+@test "install akeyless csi provider" {
+   # create the ns akeyless
+   kubectl create ns akeyless
+   # install the akeyless provider using the helm charts
+
+   helm repo add akeyless https://akeylesslabs.github.io/helm-charts
+   helm repo update
+   helm install akeyless akeyless/akeyless-csi-provider --namespace=akeyless --set csi.image.pullPolicy=Always
+
+   # wait for akeyless and akeyless-csi-provider pods to be running
```

Comment:
What are the components that helm chart installs? Is there a akeyless instance and provider?

Reply:
Akeyless CSI provider. We don't have "instance" like Vault, since we are cloud secret manager